infra/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+version: '3'
+
+services:
+  cometbft-node0:
+    container_name: cometbft-node0
+    image: "cometbft/cometbft:v0.37.x"
```

Comment:
You can use a `.env` file to put things like these versions and not have to repeat them across all the containers, see [here](https://docs.docker.com/compose/environment-variables/set-environment-variables/).

Reply:
Fixed